qemu/tests/cfg/hotplug_mem.cfg

```diff
@@ -61,12 +64,25 @@
             setup_hugepages = yes
             # mem path should be the hugpage path
             mem-path = /mnt/kvm_hugepage
-            # default pagesize is 2M, 2G guest memory need to allocate
-            # in hugepage, so page nubmer is:
-            #    target_hugepages = size_mem / page_size + 10
-            target_hugepages = 1034
             pre_command = "echo 3 > /proc/sys/vm/drop_caches"
             pre_command_noncritical = yes
+            variants:
+                - @2M_hugepage:
+                    # default pagesize is 2M, 2G guest memory need to allocate
+                    # in hugepage, so page nubmer is:
+                    #    target_hugepages = (size_mem / page_size) * 2 + 10
+                    target_hugepages = 2058
+                    expected_hugepage_size = 2048
+                - 1G_hugepage:
+                    # Warning:
+                    #     Please don't forget to update host kernel line to enable
+                    # 1G hugepage support before lanuch this test, else test will
+                    # failed.
+                    target_hugepages = 5
+                    expected_hugepage_size = 1048576
+                    only one
+                    only guest_reboot
+                    only no_policy
```

Comment:
could you help explain why only no_policy and where is the policy_mem defined (I have not found policy_mem defined)?